myhoard/restore_coordinator.py

```diff
@@ -397,7 +397,7 @@ def apply_binlogs(self):
             ranges = list(build_gtid_ranges(read_gtids_from_log(file_name, read_until_time=self.target_time)))
             if ranges:
                 last_range = ranges[-1]
-                until_after_gtids = "{}:{}".format(last_range["server_uuid"], last_range["end"])
+                until_after_gtids = f"""{last_range["server_uuid"]}:{last_range["end"]}"""
```

Comment:
The original version was probably easier to read and I think `'` would be better substitute than `"""`.